### PR TITLE
Disable connman service and pass correct dbus socket for when systemd…

### LINKED ISSUE #26: Need environment variable to work with systemd
When systemd is enabled (`ENV INITSYSTEM on`), the dbus socket will be overwritten by the container's dbus. We need to start the node app with `DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host_run/dbus/system_bus_socket`. We also need to `RUN systemctl disable connman` in Dockerfile.template.

### PR DESCRIPTION
**Pull request by pcarranzav**
… is enabled.
Should fix #26

### CODE REVIEW

**Comment by lorenzo-stoakes** [comment]
LGTM, pending tests!

File: start [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
-npm start
+if [[ "$INITSYSTEM" == "on" ]]; then
+	DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host_run/dbus/system_bus_socket npm start
```
Comment (Page-) [inline start]: I think this can be `export DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host_run/dbus/system_bus_socket` and then just one `npm start` below

Comment (pcarranzav) [inline start]: Yeah, but I wanted to keep this env var assignment exclusive to the node app.

Comment (Page-) [inline start]: Ok

**Pull request merged**